README.md

```diff
@@ -1 +1,28 @@
-# Portfolio-project
\ No newline at end of file
+# Portfolio-project
+
+A mobile portfolio page
+## Built With
+
+-HTML & CSS
+
```

Comment:
- [ ] :wave: good job with the readme but i suggest you make it more descriptive please use [this template](https://github.com/microverseinc/readme-template)